sites/main-site/src/content/blog/2024/tools-3_0_0.md

```diff
@@ -0,0 +1,252 @@
+
+- validationFailUnrecognisedParams
+- validationLenientMode
+- validationSchemaIgnoreParams
+- validationShowHiddenParams
+- validate_params
+
+Instead, use the `validation` scope for `nf-schema` options.
+
+:::note
+The plugins definition and `validation` scope have been moved after the `manifest` scope to allow access to `manifest` variables for help message customisation.
+:::
+
+The `UTILS_NFVALIDATION_PLUGIN` subworkflow has been replaced by `UTILS_NFSCHEMA_PLUGIN`, changing how the input samplesheet is read. See [the documentation](https://nextflow-io.github.io/nf-schema/2.0/migration_guide/#__tabbed_2_2) for details on using the new `samplesheetToList()` function.
+
+# Removing for-loops and try/catch Blocks from nextflow.config
+
+To prepare for upcoming Nextflow changes, code in config files has been simplified:
+
+- nf-core configs are now included without try/catch blocks
+- Include statements have been moved after profile definitions to ensure correct profile overriding
+- For more information, see these Nextflow issues: [#1792](https://github.com/nextflow-io/nextflow/issues/1792) and [#5306](https://github.com/nextflow-io/nextflow/issues/5306)
+
+# Common Merge Conflicts and Solutions
+
+## README.md
+
+In the `## Credits` section, you might see conflicts like this:
+
+```diff title="README.md"
+<<<<<<< HEAD
+- nf-core/mag was written by [Hadrien Gourlé](https://hadriengourle.com) at [SLU](https://slu.se), [Daniel Straub](https://github.com/d4straub) and - [Sabrina Krakau](https://github.com/skrakau) at the [Quantitative Biology Center (QBiC)](http://qbic.life). [James A. Fellows Yates](https://github.- com/jfy133) and [Maxime Borry](https://github.com/maxibor) at the [Max Planck Institute for Evolutionary Anthropology](https://www.eva.mpg.de) joined in version 2.2.0.
+-
+- Other code contributors include:
+-
+- - [Antonia Schuster](https://github.com/AntoniaSchuster)
+- - [Alexander Ramos](https://github.com/alxndrdiaz)
+- - [Carson Miller](https://github.com/CarsonJM)
+- - [Daniel Lundin](https://github.com/erikrikarddaniel)
+- - [Danielle Callan](https://github.com/d-callan)
+- - [Gregory Sprenger](https://github.com/gregorysprenger)
+- - [Jim Downie](https://github.com/prototaxites)
+- - [Phil Palmer](https://github.com/PhilPalmer)
+- - [@willros](https://github.com/willros)
+-
+- Long read processing was inspired by [caspargross/HybridAssembly](https://github.com/caspargross/HybridAssembly) written by Caspar Gross [@caspargross](https://github.com/caspargross)
+- nf-core/mag was written by [Hadrien Gourlé](https://hadriengourle.com) at [SLU](https://slu.se), [Daniel Straub](https://github.com/d4straub) and [Sabrina Krakau](https://github.com/skrakau) at the [Quantitative Biology Center (QBiC)](http://qbic.life).
+=======
++ nf-core/mag was originally written by Hadrien Gourlé, Daniel Straub, Sabrina Krakau, James A. Fellows Yates, Maxime Borry.
+>>>>>>> TEMPLATE
+```
+
+### Resolution
```

Comment:
I think ditch these subheadings - makes it harder to see delineation between different sections. Also makes the ToC very long.

Content in each section is short so I'm not sure that they're needed.

Reply:
I link the linkabilty of them, but agree on the ToC messiness. fixed that